app/src/main/java/br/com/fundatec/fundatecheros/home/views/profile/view/PofileActivity.kt

```diff
@@ -0,0 +1,100 @@
+package br.com.fundatec.fundatecheros.home.views.profile.view
+
+import android.os.Bundle
+import android.util.Patterns
+import androidx.appcompat.app.AppCompatActivity
+import br.com.fundatec.fundatecheros.databinding.ActivityProfileBinding
+import com.google.android.material.snackbar.Snackbar
+
+class PofileActivity : AppCompatActivity() {
```

Comment:
Possuímos um typo aqui
```suggestion
class ProfileActivity : AppCompatActivity() {
```